src/kvirc/sparser/KviIrcServerParser_literalHandlers.cpp

```diff
@@ -1787,11 +1787,12 @@ void KviIrcServerParser::parseLiteralNick(KviIrcMessage *msg)
 			old->mergeQuery(q);
 			g_pMainWindow->closeWindow(q); // deleted path
 			if(!msg->haltOutput())
-					old->output(KVI_OUT_NICK,__tr2qs("\r!n\r%Q\r [%Q@\r!h\r%Q\r] is now known as \r!n\r%Q\r"),
-					&szNick,&szUser,&szHost,&szNewNick);
+			old->output(KVI_OUT_NICK,__tr2qs("\r!n\r%Q\r [%Q@\r!h\r%Q\r] is now known as \r!n\r%Q\r"),&szNick,&szUser,&szHost,&szNewNick);
```

Comment:
the only option here is to
```	
 			if(!msg->haltOutput())
				old->output(KVI_OUT_NICK,__tr2qs("\r!n\r%Q\r [%Q@\r!h\r%Q\r] is now known as \r!n\r%Q\r"),&szNick,&szUser,&szHost,&szNewNick);
```
since it was only a wrapped line but as is without this its definitely wrong